templates/bake/Model/entity.twig

```diff
@@ -25,8 +25,11 @@
 {%- set accessible = Bake.getFieldAccessibility(fields, primaryKey) %}
 
 {%- set generatedProperties = [] %}
-{%- set uses = fileBuilder.uses(['Cake\\ORM\\Entity']) %}
-{{ element('Bake.file_header', {namespace: fileBuilder.namespace, uses: uses}) }}
+{{ element('Bake.file_header', {
+        namespace: fileBuilder.namespace,
+        uses: fileBuilder.uses(['Cake\\ORM\\Entity']),
```

Comment:
Should this have `use`?

Reply:
No. This is passing an array of imports generated by code to FileBuilder which constructs  the use statements with the user-defined imports merged in.